resolwe_runtime_utils.py

```diff
@@ -42,6 +42,14 @@ def save(key, value):
     return json.dumps({key: _get_json(value)})
 
 
+def export(*args):
```

Comment:
Hmm..., is there any reason this function is not just:

```python
def export(arg):
    """Prepend given parameter with ``export``"""
    return "export {}".format(arg)
```

Python will automatically raise a `TypeError` if the number of arguments won't match. For example:
```
$ re-export foo bar
Traceback (most recent call last):
  File "/home/tadej/.virtualenvs/resolwe-runtime-utils/bin/re-export", line 9, in <module>
    load_entry_point('resolwe-runtime-utils', 'console_scripts', 're-export')()
  File "/home/tadej/Genialis/resolwe-runtime-utils/resolwe_runtime_utils.py", line 283, in _re_export_main
    _re_generic_main(export)
  File "/home/tadej/Genialis/resolwe-runtime-utils/resolwe_runtime_utils.py", line 275, in _re_generic_main
    print(fn(*sys.argv[1:]))
TypeError: export() takes 1 positional argument but 2 were given
```